Add troubleshooting information

lukeapage opened the pull request:
The following would have saved me several hours...

sapegin commented on Readme.md:
```diff
@@ -525,6 +525,21 @@ The changelog can be found in the [Changelog.md](Changelog.md) file.
+#### Using the node engine
+
+* Certain SVG's are not supported. See the [svg2ttf](https://github.com/fontello/svg2ttf) project which is used to convert from SVG to TTF (which is then converted forward to WOFF and WOFF2).
+* `autoHint` also adjusts the font file anc can cause your font to look different to the SVG so try switchining it off.
```
anc = and? But disabling autoHint could also make icons look terrible on Windows.

sapegin commented:
That’s very useful, thank you very much!

Related issue: #221.

lukeapage commented:
Fixed spelling, added note about windows. For me (I'm on windows) it made many of my icons have corrupted paths, but that may be down to something else.

sapegin commented:
:+1:
